### PR TITLE
School attendance zone weighting file

### PR DESCRIPTION
**Pull request by maggiesu0725**
Block to school zone correspondance file

### CODE REVIEW

**Comment by maggiesu0725** [comment]
@ptatian  this file tells me the macro is not running properly

**Comment by ptatian** [comment]
@maggiesu0725 Looking at this now.

**Comment by ptatian** [comment]
@maggiesu0725 The macro _%Octo_seniorhigh()_ does not seem to exist. Did you create it somewhere?

**Comment by ptatian** [comment]
@maggiesu0725 All this macro would do is create whatever variable you want to use for the new geography from a variable in the shapefile. So, for this, you don't really need to create a macro. You could just put that code directly in your data step.

File: Prog/Weights/Wt_tr10_seniorhigh.sas [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+** Define libraries **;
+%DCData_lib( Schools )
+%DCData_lib( Census )
+
+%Calc_weights_from_blocks( 
+  geo1 = Geo2010,
+  geo2 = seniorhigh,
+  geo2check=n,
+  geo2suf=_hs
```
Comment (ptatian) [inline Prog/Weights/Wt_tr10_seniorhigh.sas]: You need to have commas (,) to separate each macro parameter.
```
...
geo2suf=_hs,
geo2name=seniorhigh,
geodlbl= Senior High School Attendance Zone,
...
```

**Review by ptatian — COMMENTED** [review]
This should fix the problem.

**Pull request merged**